app/src/main/java/org/simple/clinic/home/patients/links/PatientsTabLinkEvent.kt

```diff
@@ -0,0 +1,26 @@
+package org.simple.clinic.home.patients.links
+
+import android.Manifest
+import org.simple.clinic.activity.permissions.RequiresPermission
+import org.simple.clinic.facility.Facility
+import org.simple.clinic.platform.util.RuntimePermissionResult
+import org.simple.clinic.widgets.UiEvent
+import java.util.Optional
+
+sealed class PatientsTabLinkEvent : UiEvent
+
+data class CurrentFacilityLoaded(
+    val facility: Facility
+) : PatientsTabLinkEvent()
+
+object MonthlyScreeningReportsClicked : PatientsTabLinkEvent() {
+  override val analyticsName = "Link: Monthly Screening Report clicked"
+}
+
+data class DownloadPatientLineListClicked(
```

Comment:
Does this mean `Download IHCI patient register` was clicked?
![image](https://user-images.githubusercontent.com/9414539/222049365-c67563e6-77b8-421a-9a28-94d7ceadf6d0.png)